apps/edxapp/templates/cms/dc_worker_default_priority.yml.j2

```diff
@@ -0,0 +1,7 @@
+{% extends "dc.yml.j2" %}
+{% block command %}
+        command:
+          - "/bin/bash"
+          - "-c"
+          - python manage.py lms celery worker --loglevel=info --queues=edx.lms.core.high --hostname=edx.lms.core.default.%%h --concurrency=1
```

Comment:
I am not familiar with this command, but: it seems that the hostname matches the default host, but you are always using the `edx.lms.core.high` queue (the same remark applies to other worker templates). Is that expected?

Reply:
As discussed this morning, if you want to make this explicit, the right way to do it would be to declare the following settings explicitly in Arnold from Ansible variables: https://github.com/openfun/fun-platform/blob/master/config/lms/docker_run_production.py#L64